bin/test-ts

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo ./node_modules/.bin/mocha --exit -r ./node_modules/ts-node/register test/**/test.ts
```

Comment:
this can probably just be put into package.json

Reply:
or maybe not?

Reply:
I moved this away from package.json to avoid duplicating the same command in package.json's `test-ts` an `test-coverage`  commands.  

But I can move it back if you're fine with the little duplication

Reply:
yeah if you can even give it a slightly different command name that would be great. I would like the relevant commands to be centralized

Reply:
renamed this file to `bin/run-tests-cmd`